ql/src/java/org/apache/hadoop/hive/ql/txn/compactor/Cleaner.java

```diff
@@ -279,10 +292,14 @@ private void clean(CompactionInfo ci, long minOpenTxnGLB, boolean metricsEnabled
       }
     }
   }
+  
+  private interface CleanerTask<T> {
```

Comment:
Why do we need this specific interface instead of `Callable`?

Reply:
not really, good catch!

Reply:
fixed